### ISSUE TITLE
Update sumopy to use wxWidgets 3.0

### ISSUE THREAD

**Issue by patmalcolm91** [eclipse-sumo/sumo#3765]
Currently the sumopy tool uses the wxWidgets version 2.8, which is outdated and not available in many distributions' repos (nor are some of its dependencies). This makes it difficult for users to install and get sumopy working correctly. It would be good to update sumopy to use wxWidgets version 3.0 to eliminate this issue.

**Comment by behrisch** [comment]
@schwoz Can you comment on this?

**Comment by schwoz** [comment]
I agree this is an urgent  issue and I'll put it on my priority list. I have not upgraded yet to xwWidgets 3.0,
as there is (among others) a compatibility problem with PyOpenGL3 (the gluUnProject function returns an error). I have not followed the bugfixes but I'll have a try soon.



P.S. This a workaround that worked for me, see 
https://askubuntu.com/questions/789302/install-python-wxgtk2-8-on-ubuntu-16-04

$ echo "deb http://archive.ubuntu.com/ubuntu wily main universe" | sudo tee /etc/apt/sources.list.d/wily-copies.list
$   sudo apt update

$ sudo apt install python-wxgtk2.8

$  sudo rm /etc/apt/sources.list.d/wily-copies.list
$    sudo apt update

**Comment by patmalcolm91** [comment]
I had noticed the error from gluUnProject, as after opening this issue I attempted to change the wxWidgets version myself and chase down the errors. Unfortunately when I hit this error, my python skills were not enough to figure it out.

Thanks for the workaround. I will give it a try in the meantime.
